### ISSUE TITLE
Error pad.switchToPad is not a function

### ISSUE THREAD

**Issue by bonnebulle** [Gared/ep_bookmark#5]
Sadly I have an error :
pad.switchToPad is not a function

```[2023-03-10 16:20:56.479] [WARN] client - TypeError: pad.switchToPad is not a function -- {
  errorId: 'HKri6IiREKsIwUus4aeO',
  type: 'Uncaught exception',
  msg: 'TypeError: pad.switchToPad is not a function',
  url: 'https://pad.bonnebulle.xyz/p/testicule',
  source: 'javascript:pad.switchToPad("testonsdonc")',
  linenumber: 1,
  userAgent: 'Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/110.0',
  stack: '@javascript:pad.switchToPad("testonsdonc"):1:5\n'
}```
Thanks

**Comment by Gared** [comment]
Okay... this error happens since October 2021. I will fix this! Thanks for reporting this issue

**Comment by Gared** [comment]
Fixed in version 1.3.0
